### DISCUSSION TITLE
My account stolen

### DISCUSSION THREAD

**Discussion by ghost** [community/community#137681] [Accessibility] [answered]
### Select Topic Area

General

### GitHub Feature Area

Actions

### Body

All emails and github also

**Comment by queenofcorgis** [comment, marked as answer]
Hi , Thanks for reaching out! 

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)  As a reminder, GitHub Support can only provide support for GitHub itself. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

For your email and anything beyond GitHub, we'd recommend contacting a local IT specialist for assistance if you think your devices are affected by viruses or malware. They should be able to help you with these applications!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

_Please note: You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._
